gpt_index/readers/chroma.py

```diff
@@ -20,8 +20,11 @@ class ChromaReader(BaseReader):
     def __init__(
         self,
         collection_name: str,
-        persist_directory: str,
+        host: str = "localhost",
+        port: int = 8000,
+        persist_directory: str = None,
```

Comment:
nit: would persist_directory at the top position to preserve backwards compatibility in terms of arg ordering